website/source/api/index.html.md

```diff
@@ -77,6 +77,47 @@ to the supplied maximum `wait` time to spread out the wake up time of any
 concurrent requests. This adds up to `wait / 16` additional time to the maximum
 duration.
 
+### Implementation Details
+
+While the mechanism is relatively simple to work with, there are a few edge 
+cases that must be handled correctly.
+ * **Reset the index if it goes backwards**. While indexes in general are 
+   monotonically increasing, there are several real-world scenarios in 
```

Comment:
Can we link to a definition for "monotonically"?

Reply:
How about: "While indexes in general are monotonically increasing (i.e. they should only ever increase as time passes), "